index.html

```diff
@@ -5,10 +5,40 @@
 
 {% include banner.html %}
 
-<div class="home container">
+<div class="home">
     <div class="row justify-content-center intro">
         <div class="col-12 col-md-6 text-center py-6 overview">
-            <p>TBD</p>
+            <p class="lead">Empowering women in their tech careers through education, mentorship, community building, and career services is our mission. We provide workshops and events, connect members with industry mentors, foster a supportive community through meetups and conferences, and raise awareness for more inclusive industry practices.</p>
+        </div>
+    </div>
+
+    <div class="row justify-content-center bg-purple">
+        <div class="col-12 col-md-6 text-center pt-6 pb-4 overview">
+            <h2>Opportunities & Programmes</h2>
+            <p>Discover a world of possibilities within our community. From networking events to mentorship programs and leadership opportunities, there's something for everyone to explore. Whether you're looking to sharpen your skills, expand your professional network, or take the next step in your career, we offer a wide range of opportunities to help you achieve your goals. Join us on this journey of growth and empowerment, and unlock your full potential in the tech industry.</p>
+        </div>
+        <div class="col-12 col-md-8 text-center pb-6 programs">
+            <div class="row">
+                <div class="col">
+                    <a href="">Mentorship</a>
```

Comment:
@dricazenck I need your help with linking these a tags to the correct program pages :)

Reply:
leave to me and I will add that! 👍

Reply:
I will suggest the names you can add and I can create next the empty pages ;)